tox.ini

```diff
@@ -249,28 +249,29 @@ passenv = DOCKER_*
 [testenv:integration]
 commands =
     {[base]pip_install} acme . certbot-nginx certbot-ci
-    pytest {toxinidir}/certbot-ci/certbot_integration_tests \
+    pytest --pyargs certbot_integration_tests \
         --acme-server={env:ACME_SERVER:pebble} \
         --cov=acme --cov=certbot --cov=certbot_nginx --cov-report= \
         --cov-config={toxinidir}/certbot-ci/certbot_integration_tests/.coveragerc
     coverage report --include 'certbot/*' --show-missing --fail-under=67
     coverage report --include 'certbot-nginx/*' --show-missing --fail-under=74
 passenv = DOCKER_*
+setenv = {[testenv]setenv}
```

Comment:
I don't think we need this because the value should be inherited from `[testenv]`. See https://tox.readthedocs.io/en/latest/config.html#tox-environment-settings.

Reply:
Indeed, fixed.